main.go

```diff
@@ -75,7 +81,13 @@ func main() {
 func ParseArgs() (string, []string) {
 
 	// Print usage if no arguments are provided
-	if len(os.Args) < 2 {
+	if os.Args[1] == "version" || os.Args[1] == "-v" || os.Args[1] == "--ver" {
+		if len(os.Args) != 2 {
+			Help("version")
+		}
+
+		return "version", os.Args
+	} else if len(os.Args) < 2 {
```

Comment:
Should place this if condition at the top, otherwise the command will return `index out of rage` error when no argument is provided.

Reply:
good catch!